contrib/dist/linux/README

```diff
@@ -36,8 +36,53 @@ one or more RPMs from this tarball:
    - openmpi-devel
    - openmpi-docs
 
-The prefix, target architecture, and choice of RPM(s) to build are all
-currently hard-coded in the beginning of the buildrpm.sh script.
+The folowing arguments could be used to affect script behaviour.
+Please, do NOT set the same settings with parameters and config vars.
+
+-b
+   If you specify this option, all-in-one binary RPM would be created
+   instead of source RPM (SRPM). This option is necessary for other
+   parameters to work, as they won't be influence SRPM package.
+
+-n <name>
+   This option will change the name of the produced RPM to the <name>.
+   It is useful to use with "-o" and "-m" options if you want to have
+   multiple OpenMPI versions installed simultaneously in the same
+   enviroment.
+
+-o
+   With this option the install path of the binary RPM will be changed
+   to /opt/<name>/<version>
+
+-m
+   RPM would also install modulefiles to location specified in RPM
+   spec file.
+
+-d
+   Script will also build debuginfo RPM.
+
+-u
+   Build RPM with usNIC support. It will include all the recommended
+   compile options for usNIC as well.
+
+-f <lf_location>
+   Include support for LibFabric. <lf_location> is path of the LibFabric
```

Comment:
I am sorry. Fixed. It just looks better in my opinion 😄 (same with previous suggestion for Open MPI name).